controllers/replica_set_controller.go

```diff
@@ -301,6 +301,10 @@ func (r *ReplicaSetReconciler) ensureTLSResources(mdb mdbv1.MongoDBCommunity) er
 		if err := ensureTLSSecret(r.client, mdb); err != nil {
 			return errors.Errorf("could not ensure TLS secret: %s", err)
 		}
+		r.log.Infof("Ensuring the automation config is up to date with the latest TLS secret")
+		if _, err := r.ensureAutomationConfig(mdb); err != nil {
+			return errors.Errorf("failed to sync automation config with latest TLS secret")
+		}
```

Comment:
Do we need this call explicitly now? Q: How was the AC updated when the cert is rotated currently(in the existing cert rotation test).

Reply:
If the pods are running already it's not a problem, because they get restarted by the reconcile loop after the automation config is updated.